website/docs/docs/dbt-cloud-apis/project-state.md

```diff
@@ -0,0 +1,85 @@
+
+The applied state includes execution info, which contains metadata about how the node arrived in the applied state: the most recent execution (successful or attempted), such as when it began, its status, and how long it took.
+
+Here’s how you’d query and compare the definition  vs. applied state of a model using the Discovery API: 
+
+```graphql
+query Compare($environmentId: Int!, $first: Int!) {
+	environment(id: $environmentId) {
+		definition {
+			models(first: $first) {
+				edges {
+					node {
+						name
+						rawCode
+					}
+				}
+			}
+		}
+		applied {
+			models(first: $first) {
+				edges {
+					node {
+						name
+						rawCode 
+						executionInfo {
+							executeCompletedAt
+						}
+					}
+				}
+			}
+		}
+	}
+}
+
+```
+
+Most Discovery API use cases will favor the _applied state_ since it pertains to what has actually been run and can be analyzed.
+ 
+### State nuance per node type
+
+| Node      | Executed in DAG  | Created by execution | Exists in database | Lineage               | States               |
+|-----------|------------------|----------------------|--------------------|-----------------------|----------------------|
+| Model     | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
+| Source    | Yes              | No                   | Yes                | Downstream            | Applied & definition |
+| Seed      | Yes              | Yes                  | Yes                | Downstream            | Applied & definition |
+| Snapshot  | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
+| Test      | Yes              | Yes                  | No                 | Upstream              | Applied & definition |
+| Exposure  | No               | No                   | No                 | Upstream              | Applied & definition |
+| Metric    | No               | No                   | No                 | Upstream & downstream | Definition           |
+| Semantic model | No          | No                   | No                 | Upstream & downstream | Definition           |
+| Group     | No               | No                   | No                 | Downstream            | Definition           |
+| Macro     | Yes              | No                   | No                 | N/A                   | Definition           |
+
+ ### Caveats about state/metadata updates 
+
+Over time, Cloud Artifacts will provide information to maintain state for features/services in dbt Cloud and enable you to access state in dbt Cloud and its downstream ecosystem. It is currently focused on the latest production state, but this focus will evolve.
+
+Here are some limitations of the state representation in the Discovery API:
```

Comment:
i interpreted this paragraph as "here are the current limitations" rather than "here's how to troubleshoot" and so i am okay with the current language. 

@cafzal : what are your thoughts?

Reply:
Correct @nghi-ly these are limitations we're working to resolve over time, but let users know that they aren't bugs if they experience issues.

Reply:
got it, thank you, @cafzal !

i'll go ahead and resolve this, @runleonarun . tho, please feel free to reopen if something wasn't addressed to your satisfaction!